solutionbox/structured_data/test_mltoolbox/test_datalab_e2e.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+"""Test analyze, training, and prediction.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import json
+import logging
+import os
+import pandas as pd
+import shutil
+import six
+import sys
+import tempfile
+
+from tensorflow.python.lib.io import file_io
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '../../..')))
+
+import mltoolbox.regression.linear as reglinear  # noqa: E402
+import google.datalab.ml as dlml  # noqa: E402
+import unittest  # noqa: E402
```

Comment:
nit: this one and `e2e_functions` don't need to be here, if you move them up they won't need the `# noqa` comment